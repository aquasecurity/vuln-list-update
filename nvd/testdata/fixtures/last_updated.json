{
  "api": "2023-05-10T18:06:42.577411575Z"
}