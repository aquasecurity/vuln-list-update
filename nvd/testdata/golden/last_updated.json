{
  "api":"2023-11-28T00:00:00Z"
}